<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyForRegistrationWidgetUI</class>
 <widget class="QWidget" name="ApplyForRegistrationWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageInfo">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelInfo">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connecting to server....</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRequiredInfo">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxRequiredInfo">
         <property name="title">
          <string>Required Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="2">
           <widget class="QLabel" name="labelEmailInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelUserIDInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditEmail">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditUserID"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelEmail">
            <property name="text">
             <string>Security Email:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelUserID">
            <property name="text">
             <string>User ID:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelRetypeEmail">
            <property name="text">
             <string>Retype Email:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditRetypeEmail"/>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="labelRetypedEmailInfo">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelPasswordInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelRetypePassword">
            <property name="text">
             <string>Retype Password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditRetypePassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelRetypedPasswordInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOptionalInfo">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxOptionalInfo">
         <property name="title">
          <string>Optional Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="4" column="0">
           <widget class="QLabel" name="labelHomeAddress">
            <property name="text">
             <string>Home Address:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QComboBox" name="comboBox_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelBusinessAddress">
            <property name="text">
             <string>Business Address:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QComboBox" name="comboBox_6">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="comboBox_7">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelNickName">
            <property name="text">
             <string>Nick Name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditNickName"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelTrueName">
            <property name="text">
             <string>True Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditTrueName"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelGender">
            <property name="text">
             <string>Gender:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBoxGender">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Secret</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="labelAge">
            <property name="text">
             <string>Age:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spinBoxAge"/>
          </item>
          <item row="5" column="2" colspan="2">
           <widget class="QLineEdit" name="lineEdit_7">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="2" colspan="2">
           <widget class="QLineEdit" name="lineEdit_8">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPrevious">
       <property name="text">
        <string>&amp;Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNext">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRegister">
       <property name="text">
        <string>&amp;Register</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditUserID</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditRetypeEmail</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>lineEditNickName</tabstop>
  <tabstop>lineEditTrueName</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>spinBoxAge</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>comboBox_4</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>comboBox_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>pushButtonRegister</tabstop>
  <tabstop>pushButtonPrevious</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
